setup.py

```diff
@@ -18,38 +16,9 @@
 # This enables setuptools to install wheel on-the-fly
 SETUP_REQUIRES += ["wheel"] if "bdist_wheel" in sys.argv else []
 
-
-def make_cmdclass(basecmd):
```

Comment:
The gh-action is failing because you removed the automatic update of the template on install (which is what we talked about, indeed).

So you'll need to update the workflow file, and add a python one-liner to call explicitly your new update(). You can see that done for the wheels build because they don't accept post-install hooks.